base/applications/cleanmgr/lang/en-US.rc

```diff
@@ -0,0 +1,154 @@
+    PUSHBUTTON "View Files", IDC_VIEW_FILES, 156, 188, 60, 14, NOT WS_VISIBLE
+    ICON IDI_DRIVE, -1, 9, 8, 0, 0
+}
+
+IDD_OPTIONS_PAGE DIALOGEX 0, 0, 230, 227
+STYLE DS_CONTROL | DS_SHELLFONT | WS_CHILD | WS_CLIPCHILDREN | NOT WS_CAPTION
+FONT 8, "MS Sans Serif", 0, 0, 0
+{
+    GROUPBOX "Installed Programs", -1, 2, 9, 225, 72, BS_LEFTTEXT, WS_EX_TRANSPARENT
+    LTEXT "You can free more disk space by removing programs that you don't use.", -1, 34, 22, 187, 32
+    PUSHBUTTON "Clean up...", IDC_CLEAN_PROGRAMS, 161, 62, 60, 14
+    ICON IDI_COMPUTER, -1, 8, 22, 0, 0
+}
+
+IDD_SAGESET_PAGE DIALOGEX 0, 0, 230, 227
+STYLE DS_CONTROL | DS_SHELLFONT | WS_CHILD | WS_CLIPCHILDREN | NOT WS_CAPTION
+FONT 8, "MS Sans Serif", 0, 0, 0
+{
+    LTEXT "Check the items that you would like Disk Cleanup to automatically clean up during its scheduled run times.", -1, 37, 9, 177, 20
+    LTEXT "Files to delete:", -1, 10, 36, 65, 9
+    CONTROL "", IDC_SAGESET_LIST, "SysListView32", LVS_NOCOLUMNHEADER | LVS_ALIGNLEFT | LVS_AUTOARRANGE | LVS_SINGLESEL | LVS_REPORT | WS_BORDER | WS_TABSTOP, 8, 51, 214, 64
+    LTEXT "", IDC_STATIC_DETAILS, 14, 134, 202, 45
+    GROUPBOX "Description", -1, 8, 123, 214, 91, BS_LEFTTEXT, WS_EX_TRANSPARENT
+    ICON IDI_DRIVE, -1, 9, 8, 0, 0
+}
+
+/* Strings */
+
+STRINGTABLE
+BEGIN
+    IDS_CLEANUP "You can free up to %s of disk space on (%s):"
+    IDS_SCAN "Disk Cleanup is calculating how much space you can free on Drive (%s). This may take a few minutes to complete."
+    IDS_REMOVAL "The Disk Cleanup utility is cleaning up unnecessary files on your Drive (%s)."
+END
+
+STRINGTABLE
+BEGIN
+    IDS_DETAILS_TEMP "Program can sometimes store information in a TEMP folder. Before a program closes, it usually deletes this information.\n\nYou can safely delete temporary files that have not been modified in over a week."
+    IDS_DETAILS_RECYCLE "The Recycle Bin contains files that you have deleted from your computer. These files are temporarily removed until you empty the Recycle Bin"
+    IDS_DETAILS_CHKDSK "Old files created by ChkDsk. It is safe to remove it."
+    IDS_DETAILS_RAPPS "The RAPPS program will store the downloaded files in to a temporary folder. It is safe to remove it."
+END
+
+STRINGTABLE
+BEGIN
+	IDS_WARNING_FOLDER "Folder doesn't exist!"
+	IDS_WARNING_DRIVE "Select a drive!"
+	IDS_WARNING_OPTION "Select an option!"
+END
+
+STRINGTABLE
+BEGIN
+	IDS_CONFIRM_DELETION "Are you sure to delete these files?"
+	IDS_CONFIRM_CONFIG "Are you sure to save this configuration?"
+END
+
+STRINGTABLE
+BEGIN
+	IDS_ERROR_DRIVE "Invalid drive! Please try again!"
+END
+
+STRINGTABLE
+BEGIN
+    IDS_LABEL_CHKDSK "Old ChkDsk Files"
+    IDS_LABEL_RECYCLE "Recycle Bin"
+    IDS_LABEL_RAPPS "Temporary RAPPS Files"
+    IDS_LABEL_TEMP "Temporary Files"
+END
+
+STRINGTABLE
+BEGIN
+    IDS_CHOICE_DLG_TITLE "Disk Cleanup for (%s)"
+    IDS_START_DLG_TITLE "Disk Cleanup: Drive Selection"
```

Comment:
This is really error prone: if this is not translated exactly as the first dialog, or the user is not in the first dialog anymore, this will fail to find it.